server/services/placementService.ts

```diff
@@ -21,4 +24,17 @@ export default class PlacementService {
 
     return placementClient.assignKeyworker(premisesId, placementId, keyworkerAssignment)
   }
+
+  async getNonArrivalReasons(token: string): Promise<Array<NonArrivalReason>> {
+    const client = this.referenceDataClientFactory(token)
+    const allNonArrivalReasons = await client.getNonArrivalReasons()
+
+    return allNonArrivalReasons.filter(({ isActive }) => isActive)
```

Comment:
As discussed, the API should only ever return reference data where `isActive = true`, so we'll be able to remove this when the API is fixed for this endpoint.